descriptors/ModuleDescriptor-template.json

```diff
@@ -627,6 +627,31 @@
             "invoice.invoice-lines.item.put"
           ]
         },
+        {
+          "methods": ["PUT"],
+          "pathPattern": "/orders/pieces-batch/status",
```

Comment:
also please create PR in acq-models with new schemas

Reply:
regarding enpoint name, maybe /orders/pieces-status ?

Reply:
or idea was to support updating other fields except status in bathc, okay makes sense, so in this case lets ignore my comment